indico/modules/rb_new/controllers/backend.py

```diff
@@ -76,7 +81,16 @@ def _process(self, args):
         query = search_for_rooms(args, only_available=filter_availability)
         query = query.limit(args['limit']).offset(args['offset'])
         rooms, total = with_total_rows(query)
-        return jsonify(total=total, rooms=rooms_schema.dump(rooms).data)
+        if _cache.get('rooms-sprite') is None or _cache.get('rooms-sprite-mapping') is None:
```

Comment:
let's not read the whole image from cache here. better to assume it exists if we have a mapping